segmentation/utils/visualize_test.py

```diff
@@ -10,3 +11,17 @@ def test_visualize_z_scores():
     # Assign random phenotype titles
     pheno_titles = [chr(i) for i in range(ord('a'), ord('z') + 1)]
     plot = visualize.visualize_z_scores(z, pheno_titles)
+    
+def test_visualize_cells():
+    rand_type = []
+    ids = []
+    for x in range(0,1000):
```

Comment:
Just nitpicking here, it's OK to leave out the 0 in range in this case and just use range(1000). If only one value is specified, it is assumed that you start from 0.